index.js

```diff
@@ -40,7 +52,11 @@ const onResize = (stream, cb) => {
 	return stopListening
 }
 
-const wrap = (p) => {
+/**
+ * @param {*} p
+ * @param {boolean} v whether or not to attempt to parse vim keybindings
+ */
+const wrap = (p, v = false) => {
```

Comment:
Change this to `(p, opt = {}) => {` and do `opt = Object.assign({}, defaults, opt)` beneath.